app/views/layouts/application.html.erb

```diff
@@ -19,13 +19,18 @@
     <div class="nav-item">
       <a class="nav-link" href="/freelancers">Freelancers</a>
     </div>
-      <div class="nav-item">
-        <% if current_user.nil? %>
-        <a class="nav-link" href="/login">Login</a>
-        <% else %>
-        <a class="nav-link" href="/logout">Logout</a>
-        <% end %>
-      </div>
+    <div class="nav-item">
```

Comment:
Nice additions. We might check into whether or not there are ways to pull logic out of the navbar, but for now it's doing what we need it to do!